enable_disable_persistence.md

```diff
@@ -0,0 +1,180 @@
+# Enable/Disable State Persistence Spec
+
+## Background
+
+Tilt currently has a system for changing the arguments to `tilt up` at
+runtime. We usually call it `tilt args`.
+
+https://docs.tilt.dev/tiltfile_config.html
+
+Most teams that use this system are using it for enabling and disabling sets of
+services, as demonstrated in this example:
+
+https://docs.tilt.dev/tiltfile_config.html#run-a-defined-set-of-services
+
+This helps save CPU when you only need to run a subset of your services.
+
+A number of teams have built wrappers around `tilt args`. This has helped us to
+learn a lot about the ergonomics that people want, and the pains of the current
+system. We have mocks and a pretty good idea of how we want the narrowly-focused
+single-click enable/disable interaction to work.
+
+For more details, see [Enable/Disable Resource API Spec](./enable_disable_resources.md).
+
+But we have different ideas on how we want the enable/disable interaction to work
+across multiple Tilt runs, and how the state should persist! [^1]
+
+## Primary Goal
```

Comment:
1. I think it's totally on the table to either change the semantics of `tilt args`, or to introduce a new CLI that accomplishes similar product goals (e.g., "start tilt with project1", "start tilt with project1+3 extra services", etc)

2. Ya, I also outlined the semantics of systemd below. But I feel like saying "the semantics of docker-compose up/down would make this implementation simpler" is a bit like saying "if we rewrote Tilt in Javascript, we wouldn't have panics anymore." That's true, it would solve that problem! But I think it would also create tons of new problems (as much as i like JS)